openedx/core/djangoapps/user_api/preferences/views.py

```diff
@@ -263,3 +270,50 @@ def delete(self, request, username, preference_key):
             return Response(status=status.HTTP_404_NOT_FOUND)
 
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class ChangeToVerifiedMode(APIView):
+    """
+    Change the status to 'approved' in Software Secure Photo Verification model for the given user.
+    """
+    authentication_classes = (JwtAuthentication,)
```

Comment:
@Squirrel18 how would ecommerce authenticate against this endpoint, which configuration will we require on the ecommerce site? I'm not pretty familiar with JWT.

Reply:
@jfavellar90 no se necesita nada adicional, site tiene algo que se llama siteconfiguration y permite extraer el access token para esta petición.

Reply:
Según esta clase, solo se puede acceder por JWT, y el site_configuration no es más que un diccionario de valores almacenados para un dominio. Que quieres decir con que de ahí se puede extraer el access_token? por favor añade un link a la parte correspondiente del código que hace el llamado

Reply:
@felipemontoya  https://github.com/eduNEXT/ecommerce/blob/3ed33c613eda607b90af18035f3e3fd43c81c843/ecommerce/extensions/payment/processors/payu.py
Linea 223, ese metodo hace una peticion a oauth2/access_token/ con el client_id y el client_secret y le pides que sea un JWT.